### DISCUSSION TITLE
Implementing Azure Advisor’s Recommendation for Disabling Automounting API Credentials in Kubernetes

### DISCUSSION THREAD

**Discussion by iflach** [argoproj/argo-helm#2994] [General]
Hello everyone,

I would like to implement the Azure Advisor recommendation “Kubernetes clusters should disable automounting API credentials” during our cluster hardening process. The issue is that while I can easily set the automountServiceAccountToken parameter to false for the Pods such as RepoServer, ApplicationSet, Notifications, etc., I am unable to figure out how to manually mount the Secret in this case. Could someone please assist me with this?

Thank you!

**Comment by yu-croco** [comment]
Hi @iflach , we always welcome PRs. There was a PR for argo-cd, https://github.com/argoproj/argo-helm/pull/2625 , so I think you can refer this for argo-workflows.
